samples/linear_layout/main.go

```diff
@@ -5,15 +5,26 @@
 package main
 
 import (
+	"flag"
 	"github.com/google/gxui"
 	"github.com/google/gxui/drivers/gl"
 	"github.com/google/gxui/math"
-	"github.com/google/gxui/samples/flags"
 	"github.com/google/gxui/themes/dark"
+	"github.com/google/gxui/themes/light"
+)
+
+var (
+	flagTheme          = flag.String("theme", "dark", "Theme to use {dark|light}.")
```

Comment:
We could add to the `samples/flags` package:

```go

var flagTheme = flag.String("theme", "dark", "Theme to use {dark|light}.")

// CreateTheme creates and returns the theme specified on the command line.
// The default theme is dark.
func CreateTheme(driver gxui.Driver) gxui.Theme {
	if *flagTheme == "light" {
		return light.CreateTheme(driver)
	} else {
		return dark.CreateTheme(driver)
	}
}
```

That way we keep all the sample-specific flags in one place.